### DISCUSSION TITLE
Includes all of Tailwind 2.0?

### DISCUSSION THREAD

**Discussion by trevyn** [windicss/windicss#9] [Q&A] [answered]
This project looks very cool!

It wasn’t clear to me: does this support all of Tailwind 2.0? (It was a significant shift from 1.x and when projects just mention “Tailwind”, sometimes they’re not updated for the new release!)

And do you expect to keep it updated as new Tailwind features arrive?

windicss + Svelte looks very promising... :)

**Comment by voorjaar** [comment, marked as answer]
Although the implementation method is exactly the opposite of tailwindcss, it fully supports all utilities of tailwincss v2.0, and the next version already supports tailwind.config.js. I'll make sure that the update is synchronized with the tailwind. And sometimes may add some ideas of my own too.

I don't have time to write documents for now. I'll do it soon. when I finish writing the document, you should understand better.
